docs/general/administration/hardware-selection.md

```diff
@@ -0,0 +1,125 @@
+---
+uid: admin-hardware-selection
+title: Hardware Selection
+---
+
+# Selecting Appropriate Hardware
+
+This page covers what you need to know in order to select appropriate hardware for a Jellyfin server and take full advantage of its features (ex. Hardware Acceleration).
+
+## TL;DR
+
+For a Jellyfin server, we recommend the following:
+
+### Normal Server From Scratch
+
+- CPU: 4 Threads
+  - Please Avoid 11th Gen or older Intel Atom and Celeron / Pentium N / J series CPUs.
+  - Please Avoid Rapsberry PIs and other low powered SBCs.
+  - Please Avoid Intel Core M / Y series CPUs.
+- RAM: 8GB or more
+- Storage: SSD for Jellyfin files and transcoding cache.
+- Graphics: Intel > Nvidia > AMD > Apple.
+  - We recommend a system with Intel 7th gen or newer Core series CPU with integrated graphics.
+  - When using dedicated graphics, we recommend Intel ARC over Nvidia GTX 16 / RTX 20 or newer GPUs, excluding the GTX 1650.
```

Comment:
Might be good to add an infobox about the reasoning behind this.

Reply:
I also think this could be phrased a little bit better. This is unclear, and I think the language should be made less direct.

"You should consider Intel ARC over Nvidia" vs "We recommend Intel ARC over Nvidia"

This language would be better throughout.

Reply:
Addressed in a commit